src/app/features/dashboard_Id/Card/cardFormModals/CreateCardForm.tsx

```
@@ -0,0 +1,281 @@
+    const payload: CardFormData = {
+      ...data,
+      dashboardId: dashboardId,
+      columnId: columnId,
+      // tags: data.tags ?? [],
+      // imageUrl: data.imageUrl,
+    }
+
+    if (!data.dueDate) delete payload.dueDate
+    if (!data.imageUrl || !preview) delete payload.imageUrl // delete로 아예 필드의 해당 key를 지워야, 서버가 "없음"으로 인식함..
+    console.log('🌀', data.imageUrl)
+    console.log('submitted', payload)
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)